spec/forms/sufia/batch_edit_form_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe Sufia::Forms::BatchEditForm do
+  let(:model) { GenericWork.new }
+  let(:work1) { create :generic_work, title: ["title 1"], keyword: ["abc"], creator: ["Wilma"], language: ['en'], contributor: ['contributor1'], description: ['description1'], rights: ['rights1'], subject: ['subject1'], identifier: ['id1'], based_near: ['based_near1'], related_url: ['related_url1'] }
+  let(:work2) { create :generic_work, title: ["title 2"], keyword: ["123"], creator: ["Fred"], publisher: ['Rand McNally'], language: ['en'], resource_type: ['bar'], contributor: ['contributor2'], description: ['description2'], rights: ['rights2'], subject: ['subject2'], identifier: ['id2'], based_near: ['based_near2'], related_url: ['related_url2'] }
+  let(:batch) { [work1.id, work2.id] }
+  let(:form) { described_class.new(model, ability, batch) }
+  let(:ability) { Ability.new(user) }
+  let(:user) { build(:user, display_name: 'Jill Z. User') }
+
+  describe "#terms" do
+    subject { form.terms }
+    it { is_expected.to eq [:creator,
```

Comment:
@mjgiarlo I put date_created back. I always think that is the system date created...  The other two were already being remove. See the comment to your comment above